Mathlib/AlgebraicGeometry/Morphisms/Finite.lean

```diff
@@ -58,6 +57,11 @@ instance : ContainsIdentities @IsFinite :=
 
 instance : IsMultiplicative @IsFinite where
 
+lemma SpecMap_iff {R S : CommRingCat.{u}} (f : R ⟶ S) :
```

Comment:
I am not completely sure what the name should be here:
```suggestion
lemma specMap_iff {R S : CommRingCat.{u}} (f : R ⟶ S) :
```

Reply:
I think we call it `SpecMap` across the AG library. The reason being that `Spec` is always uppercase in the literature.